tiled/authenticators.py

```diff
@@ -5,10 +5,13 @@
 import re
 import secrets
 from collections.abc import Iterable
+from typing import Any, cast
 
 import httpx
 from fastapi import APIRouter, Request
-from jose import JWTError, jwk, jwt
+from jose import JWTError, jwk, jwt, jws
+from pydantic import Secret
+import requests
```

Comment:
Can we use `httpx` for this, since it's use for all other HTTP client stuff in Tiled?